samples/net/openthread/ncp/Kconfig

```diff
@@ -0,0 +1,15 @@
+# Config options for OpenThread Border Router sample app
+
+# Copyright (c) 2020 Tridonic GmbH & Co KG
+# SPDX-License-Identifier: Apache-2.0
+
+mainmenu "OpenThread NCP"
+
+menu "Application configuration"
+
+config APPLICATION_VERSION
```

Comment:
We have Kconfig options in other network samples, we could follow similar naming convention for these. So please add prefix `NET_SAMPLE_...` here so this could be then `config NET_SAMPLE_APPLICATION_VERSION` or `NET_SAMPLE_APP_VERSION`.

Reply:
fixed.